Navigation: Expand & Improve editorType/widget Docs (#8514)
DevExpress/devextreme-documentation: api-reference/_hidden/dxHtmlEditorToolbarItem/widget.md

        import 'devextreme/ui/tag_box';

- To update the item component's state/options at runtime, do not use this property. Configure the component within a [template]({currentpath}/#template) instead.

##### Vue

- To use this property, import the specified editor's [module](/concepts/Common/Modularity/02%20DevExtreme%20Modules%20Structure '/Documentation/Guide/Common/Modularity/DevExtreme_Modules_Structure/') from the `devextreme` folder as follows:

        import 'devextreme/ui/tag_box';

- To update the item component's state/options at runtime, do not use this property. Configure the component within a [template]({currentpath}/#template) instead.

##### React

- To use this property, import the specified editor's [module](/concepts/Common/Modularity/02%20DevExtreme%20Modules%20Structure '/Documentation/Guide/Common/Modularity/DevExtreme_Modules_Structure/') from the `devextreme` folder as follows:

        import 'devextreme/ui/tag_box';

> - To update the item component's state/options at runtime, do not use this property. Configure the component within [render]({currentpath}/#render) or [component]({currentpath}/#component) instead.

Reviewer: ```suggestion
- To update the item component's state/options at runtime, configure the component in [render]({currentpath}/#render) or [component]({currentpath}/#component) instead of using this property.
```
